### PR TITLE
Fix rendering of table of dependencies in MacDown

### PR DESCRIPTION
**Pull request by mohamed-barakat**
MacDown needs at least three dashes

### CODE REVIEW

File: templates/README.md_HEADER.j2 [hunk 1]
```diff
@@ -14,8 +14,9 @@
 ### Dependencies
 
 To obtain current versions of all dependencies, clone the following repositories:
-|   | Repository | URL |
-| - | ---------- | --- |
+
```
Comment (zickgraf) [inline templates/README.md_HEADER.j2:17]: Do we need this new empty line?

File: templates/README.md_HEADER.j2 [hunk 2]
```diff
@@ -14,8 +14,9 @@
...
+|    | Repository | URL |
+|--- | ---------- | --- |
```
Comment (zickgraf) [inline templates/README.md_HEADER.j2:19]:
```suggestion
|     | Repository | URL |
| --- | ---------- | --- |
```

**Review by zickgraf — CHANGES_REQUESTED** [review]

File: templates/README.md_HEADER.j2 (hunk 1 above)
Comment (mohamed-barakat) [reply templates/README.md_HEADER.j2:17]: Yes, my MacDown editor needs it.

File: templates/README.md_HEADER.j2 (hunk 2 above)
Comment (mohamed-barakat) [reply templates/README.md_HEADER.j2:19]: The reason is that
```
|--- |
| 1. |
```
is nicer to read in the clear text than
```
| --- |
| 1. |
```

**Pull request merged**